src/apps/explorer/ExplorerApp.tsx

```diff
@@ -110,6 +118,7 @@ const AppContent = (): JSX.Element => {
           <Route path={pathPrefix + '/'} exact component={Home} />
           <Route path={pathPrefix + '/orders/:orderId'} exact component={Order} />
           <Route path={pathPrefix + '/address/:address'} exact component={UserDetails} />
+          <Route path={pathPrefix + '/search/:orderId'} exact component={SearchNotFound} />
```

Comment:
I would name it `searchString`, as you might be searching by things other than `orderId`